packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue

```diff
@@ -67,58 +69,153 @@ export default defineComponent({
 	components: {
 		KtAvatar,
 	},
-	mixins: [clickaway],
 	props: makeProps(KottiUserMenu.propsSchema),
 	setup(props: KottiUserMenu.PropsInternal) {
+		const triggerRef = ref<HTMLElement | null>(null)
+		const userMenuRef = ref<HTMLElement | null>(null)
+
 		const isMenuShow = ref(false)
-		const isNarrow = inject<ComputedRef<boolean>>(
-			IS_NAVBAR_NARROW,
-			computed(() => false),
+		const context = inject(
+			KT_NAVBAR_CONTEXT,
+			computed(() => ({ isNarrow: false, theme: null })),
+		)
+
+		useTippy(
+			triggerRef,
+			computed(() => ({
+				animation: false,
+				appendTo: () => document.body,
+				arrow: false,
+				content: userMenuRef.value ?? undefined,
+				interactive: true,
+				maxWidth: 'none',
+				offset: [0, 0],
+				onHide: () => {
+					isMenuShow.value = false
+				},
+				onShow: () => {
+					isMenuShow.value = true
+				},
+				theme: context.value.theme
+					? `kt-usermenu-${context.value.theme}`
```

Comment:
Kinda looks like `theme` should default to something like `default` and you can get rid of the conditionals etc. around that then